Assets/HoloToolkit-Examples/GazeRuler/Scripts/LineManager.cs

```diff
@@ -1,137 +1,143 @@
-﻿using UnityEngine;
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
 using HoloToolkit.Unity;
 using System.Collections.Generic;
 using HoloToolkit.Unity.InputModule;
 
-/// <summary>
-/// mananger all lines in the scene
-/// </summary>
-public class LineManager : Singleton<LineManager>, IGeometry
+namespace HoloToolkit.Examples.GazeRuler
 {
-    // save all lines in scene
-    private Stack<Line> Lines = new Stack<Line>();
-
-    private Point lastPoint;
-
-    private const float defaultLineScale = 0.005f;
-
-    // place point and lines
-    public void AddPoint(GameObject LinePrefab, GameObject PointPrefab, GameObject TextPrefab)
+    /// <summary>
+    /// mananger all lines in the scene
+    /// </summary>
+    public class LineManager : Singleton<LineManager>, IGeometry
     {
+        // save all lines in scene
+        private Stack<Line> Lines = new Stack<Line>();
 
-        var hitPoint = GazeManager.Instance.HitPosition;
+        private Point lastPoint;
 
-        var point = (GameObject)Instantiate(PointPrefab, hitPoint, Quaternion.identity);
-        if (lastPoint != null && lastPoint.IsStart)
-        {
-            var centerPos = (lastPoint.Position + hitPoint) * 0.5f;
+        private const float defaultLineScale = 0.005f;
 
-            var directionFromCamera = centerPos - Camera.main.transform.position;
+        // place point and lines
+        public void AddPoint(GameObject LinePrefab, GameObject PointPrefab, GameObject TextPrefab)
+        {
 
-            var distanceA = Vector3.Distance(lastPoint.Position, Camera.main.transform.position);
-            var distanceB = Vector3.Distance(hitPoint, Camera.main.transform.position);
+            var hitPoint = GazeManager.Instance.HitPosition;
```

Comment:
Could we name explicit variable types for this PR?
Makes it easier for newbies and reviewers to better understand code.
Exceptions are fine for obvious types and same line creation with new keywords.